rmgpy/molecule/molecule.py

```diff
@@ -1621,4 +1621,41 @@ def toGroup(self):
         
         return group
 
+    def getAromaticSSSR(self):
+        """
+        Returns the smallest set of smallest aromatic rings
```

Comment:
You should probably mention that this is what rdkit thinks are aromatic rings and maybe make the distinction between with the two types of rings you could have in clar structures?